RecoLocalCalo/HcalRecProducers/python/hfQIE10Reco_cfi.py

```diff
@@ -0,0 +1,15 @@
+import FWCore.ParameterSet.Config as cms
+
+hfQIE10Reco = cms.EDProducer("HcalSimpleReconstructor",
+    correctionPhaseNS = cms.double(0.0),
+    digiLabel = cms.InputTag("simHcalUnsuppressedDigis","HFQIE10DigiCollection"),
```

Comment:
I had assumed that @jmmans and co. had handled this already, but now that I look, it appears that HcalPacker has not yet been modified to handle QIE10 digis. We should take care of this, but I hope it can come in a subsequent PR.